edelphi/src/main/java/fi/metatavu/edelphi/pages/admin/ManagePlanPageController.java

```diff
@@ -0,0 +1,112 @@
+    
+    String[] supportedLocales = LocalizationUtils.getSupportedLocales();
+    Plan plan = null;
+    
+    if (pageRequestContext.getBoolean("create")) {
+      LocalizedEntry name = localizedEntryDAO.create();
+      LocalizedEntry description = localizedEntryDAO.create();
+      
+      for (String supportedLocale : supportedLocales) {
+        LocalizationUtils.updateText(name, LocaleUtils.toLocale(supportedLocale), messages.getText(locale, "admin.managePlan.name", new Object[] { supportedLocale }));
+        LocalizationUtils.updateText(description, LocaleUtils.toLocale(supportedLocale), messages.getText(locale, "admin.managePlan.description", new Object[] { supportedLocale }));
+      }
+      
+      plan = planDAO.create(SubscriptionLevel.PLUS, 0d, "EUR", 365, name, description, Boolean.FALSE);
+    } else {
+      Long planId = pageRequestContext.getLong("planId");
+      if (planId != null) {
+        plan = planDAO.findById(planId);
+      }
+    }
+
+    if (plan == null) {
+      throw new PageNotFoundException(locale);
+    }
+    
+    Delfoi delfoi = RequestUtils.getDelfoi(pageRequestContext);
+
+    ActionUtils.includeRoleAccessList(pageRequestContext);
+    
+    List<String> subscriptionLevels = new ArrayList<>();
+    subscriptionLevels.add(SubscriptionLevel.PLUS.name());
+    subscriptionLevels.add(SubscriptionLevel.PREMIUM.name());
+
+    Map<String, String> nameLocales = new HashMap<>();
+    Map<String, String> descriptionLocales = new HashMap<>();
+    Map<String, String> nameValues = new HashMap<>();
+    Map<String, String> descriptionValues = new HashMap<>();
+    Map<String, String> subscriptionLevelTitles = new HashMap<>();
+    
+    for (String supportedLocale : supportedLocales) {
+      nameLocales.put(supportedLocale, messages.getText(locale, "admin.managePlan.name", new Object[] { supportedLocale }));
+      descriptionLocales.put(supportedLocale, messages.getText(locale, "admin.managePlan.description", new Object[] { supportedLocale }));
+      nameValues.put(supportedLocale, LocalizationUtils.getLocalizedText(plan.getName(), LocaleUtils.toLocale(supportedLocale)));
+      descriptionValues.put(supportedLocale, LocalizationUtils.getLocalizedText(plan.getDescription(), LocaleUtils.toLocale(supportedLocale)));
+    }
+    
+    for (String subscriptionLevel : subscriptionLevels) {
+      subscriptionLevelTitles.put(subscriptionLevel, messages.getText(locale, String.format("generic.subscriptionLevels.%s", subscriptionLevel)));
+    }
+    
+    pageRequestContext.getRequest().setAttribute("delfoi", delfoi);
+    pageRequestContext.getRequest().setAttribute("subscriptionLevels", subscriptionLevels.toArray(new String[0]));
+    pageRequestContext.getRequest().setAttribute("locales", supportedLocales);
+    pageRequestContext.getRequest().setAttribute("nameLocales", nameLocales);
+    pageRequestContext.getRequest().setAttribute("descriptionLocales", descriptionLocales);;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)